rotkehlchen/api/v1/resources.py

```diff
@@ -2568,7 +2569,7 @@ def get(self) -> Response:
 
 
 class AddressbookResource(BaseMethodView):
-    post_delete_schema = AddressbookAddressesSchema()
+    post_delete_schema = OptionalAddressesListSchema()
```

Comment:
Nope, this will break (and by checking the tests it has) all address book tests.

Reply:
Yeah, yeah